src/main/java/eu/eutampieri/catacombs/model/SimpleWeapon.java

```diff
@@ -84,7 +84,7 @@ public String getName() {
      * @param delta time between updates
      */
     @Override
-    public void update(final int delta) {
+    public void update(final float delta) {
```

Comment:
Can we just use `int`s for time deltas? Floats are bad

Reply:
Sure